openerp/addons/base/migrations/7.0.1.3/pre-migration.py

```diff
@@ -128,6 +128,13 @@ def update_base_sql(cr):
             "ALTER TABLE ir_act_client INHERIT ir_actions")
 
 
+def is_module_installed(cr, module):
```

Comment:
you can remove this as it's merges in https://github.com/OCA/openupgradelib/pull/20
don't forget to import it though